### PR TITLE
Apply Doctrine CS 5.0

### PR DESCRIPTION
**Pull request by Majkl578**
|      Q       |   A
|------------- | -----------
| Type         | improvement
| BC Break     | no

### CODE REVIEW

File: lib/Doctrine/Migrations/Tools/Console/Command/DiffCommand.php [hunk 1]
```diff
@@ -72,8 +72,7 @@ protected function configure() : void
                 InputOption::VALUE_OPTIONAL,
                 'Max line length of unformatted lines.',
                 120
-            )
-        ;
+            );
```
[the remark this answers is not in the document]
Comment (Majkl578) [reply lib/Doctrine/Migrations/Tools/Console/Command/DiffCommand.php]: Yep.

File: tests/Doctrine/Migrations/Tests/BoxPharCompileTest.php [hunk 2]
```diff
@@ -41,7 +41,7 @@ public function testCompile() : void
 
         $process = new Process($runDoctrinePharCommand);
 
-        $process->start(function ($type, $buffer) use (&$output, &$successful) : void {
+        $process->start(static function ($type, $buffer) use ($successful) : void {
```
[the remark this answers is not in the document]
Comment (Majkl578) [reply tests/Doctrine/Migrations/Tests/BoxPharCompileTest.php]: Well spotted, I missed this one. Looks like a bug, will investigate once back on PC.

[the remark this answers is not in the document]
Comment (Majkl578) [reply tests/Doctrine/Migrations/Tests/BoxPharCompileTest.php]: Fixer bug reported here: https://github.com/slevomat/coding-standard/issues/500, fixed fix manually.
Also removed unused `$buffer` which was not detected by any sniff. :/

**Comment by jwage** [comment]
Should we get https://github.com/doctrine/migrations/pull/749 merged?

**Comment by Majkl578** [comment]
Let's do this one first, I'll then rebase #749.

**Comment by jwage** [comment]
You da man 👍

**Pull request merged**